apps/lend/package.json

```diff
@@ -27,25 +27,24 @@
   },
   "dependencies": {
     "@curvefi/lending-api": "^2.4.0",
-    "@lingui/detect-locale": "^4.6.0",
-    "@lingui/react": "^4.6.0",
+    "@lingui/cli": "^4.6.0",
```

Comment:
If it worked with dev dependencies, why move it here?

Reply:
Ah cheers for noticing. I did some shuffling around and moved those dev dependencies out, only to realize they couldn't. So I moved them back, but did not notice it wasn't dev dependencies. They're moved back now.